client/jetpack-cloud/sections/partner-portal/stored-credit-card-v2/credit-card-actions.tsx

```diff
@@ -1,3 +1,4 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
```

Comment:
It is recommended to always import like below and use the Middleware:

```
import { recordTracksEvent } from 'calypso/state/analytics/actions';
```

Note: [Unless you have a strong reason to call `recordTracksEvent` directly, you should use the Analytics Middleware instead](https://github.com/Automattic/wp-calypso/blob/2544768a56d3a10a77cacf8c1bff65ab78fb6881/packages/calypso-analytics/README.md)